modules/features2d/src/affine_feature.cpp

```diff
@@ -0,0 +1,353 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
```

Comment:
Please add OpenCV short license header above with note that:

"This file is based on code issued with the following license"